gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/grpc/v4/GrpcServerStreamingV4IntegrationTest.java

```diff
@@ -43,12 +42,13 @@
 @GatewayTest
 @DeployApi({ "/apis/v4/grpc/streaming-greeter.json" })
 @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@Timeout(value = 10, timeUnit = TimeUnit.SECONDS)
```

Comment:
I wonder if we should not go up to 30 seconds (default jUnit timeout) since we noticed some high resource usage on CI runs